crates/pathfinder/src/bin/pathfinder.rs

```diff
@@ -9,7 +9,7 @@ async fn main() {
     // TODO: get database path from configuration
     let storage = Storage::migrate("database.sqlite".into()).unwrap();
     // TODO: pick the correct sequencer based on the Ethereum chain.
-    let sequencer = sequencer::Client::goerli().unwrap();
+    let sequencer = sequencer::Client::new(pathfinder_lib::ethereum::Chain::Goerli).unwrap();
```

Comment:
```suggestion
    let sequencer = sequencer::Client::new(crate::ethereum::Chain::Goerli).unwrap();
```

I'm surprised the crate name even worked :)

Reply:
Nevermind this is in example. Looked familiar and I confused it :D